sed/config/NXmpes_config-HEXTOF.json

```diff
@@ -8,9 +8,10 @@
   "/ENTRY[entry]/experiment_institution": "Deutsches Elektronen-Synchrotron DESY",
   "/ENTRY[entry]/experiment_facility": "Free-Electron Laser FLASH",
   "/ENTRY[entry]/experiment_laboratory": "@attrs:metadata/creationLocation",
-  "/ENTRY[entry]/entry_identifier": "@attrs:metadata/_id",
+  "/ENTRY[entry]/entry_identifier": "@attrs:metadata/pid",
   "/ENTRY[entry]/USER[user]": {
     "name": "@attrs:metadata/principalInvestigator",
+    "affiliation": "t.b.d.",
```

Comment:
The affiliation should be provided to uniquely identify a person. Can this be obtained from SciCat?

Reply:
This is still in the status to be filled manually, but as soon as DESY moves to the new version of SciCat this field should be added automatically from the beamtime metadata file.

Reply:
You can add additional metadata by providing a dictionary, as done in example 3. Additionally, this can be read off from a yaml file. Maybe use one of these approaches as demonstration for missing metadata at the moment.

Reply:
In this particular case, if I am trying to add missing information as a manual metadata following tutorial 3
```
metadata = {}
metadata['principalInvestigator'] = {}
metadata['principalInvestigator']['affiliation'] = 'Deutsches Elektronen-Synchrotron DESY'
metadata['principalInvestigator']['email'] = 'dmytro.kutnyakhov@desy.de'
```
I am getting an error message:
ValueError: Cannot merge dictionary with non-dictionary entry!

For sure, I can add this info as e.g. a new key [user] or so, and then adjust config file accordingly using information from two different keys
```
"/ENTRY[entry]/USER[user]": {
    "name": "@attrs:metadata/principalInvestigator",
    "affiliation": "@attrs:metadata/user/affiliation",
```
but this way probably would a bit misleading.

Reply:
Well, as the entry "principalInvestigator" is given as string by SciCat, you obviously cannot merge it with a dict, it can either be a dict or a string. 
If you don't like the option you sketched below (which I would find okay until these details are also in scicat), you can change the that gathers the data from Scicat, and rewrite the prinicpalInvestigator entry to be a dict, and the given string as "name" in that dict.